CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Change log
 life-sync uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+# 1.0.2
+
+* Branch as newtype.
```

Comment:
Could you please add the issue number with the link in the following format:

```
* [#42](https://github.com/kowainik/life-sync/issues/42):
  Create newtype `Branch`.
```

Reply:
Done.